%!TEX TS-program = xelatex
%!TEX encoding = UTF-8 Unicode
% Awesome CV LaTeX Template for CV/Resume
%
% This template has been downloaded from:
% https://github.com/posquit0/Awesome-CV
%
% Author:
% Claud D. Park <posquit0.bj@gmail.com>
% http://www.posquit0.com
%
%
% Adapted to be an Rmarkdown template by Mitchell O'Hara-Wild
% 23 November 2018
%
% Template license:
% CC BY-SA 4.0 (https://creativecommons.org/licenses/by-sa/4.0/)
%
%-------------------------------------------------------------------------------
% CONFIGURATIONS
%-------------------------------------------------------------------------------
% A4 paper size by default, use 'letterpaper' for US letter
\documentclass[11pt,a4paper,]{awesome-cv}

% Configure page margins with geometry
\usepackage{geometry}
\geometry{left=1.4cm, top=.8cm, right=1.4cm, bottom=1.8cm, footskip=.5cm}


% Specify the location of the included fonts
\fontdir[fonts/]

% Color for highlights
% Awesome Colors: awesome-emerald, awesome-skyblue, awesome-red, awesome-pink, awesome-orange
%                 awesome-nephritis, awesome-concrete, awesome-darknight

\definecolor{awesome}{HTML}{414141}

% Colors for text
% Uncomment if you would like to specify your own color
% \definecolor{darktext}{HTML}{414141}
% \definecolor{text}{HTML}{333333}
% \definecolor{graytext}{HTML}{5D5D5D}
% \definecolor{lighttext}{HTML}{999999}

% Set false if you don't want to highlight section with awesome color
\setbool{acvSectionColorHighlight}{true}

% If you would like to change the social information separator from a pipe (|) to something else
\renewcommand{\acvHeaderSocialSep}{\quad\textbar\quad}

\def\endfirstpage{\newpage}

%-------------------------------------------------------------------------------
%	PERSONAL INFORMATION
%	Comment any of the lines below if they are not required
%-------------------------------------------------------------------------------
% Available options: circle|rectangle,edge/noedge,left/right

\name{Xiangyu Yao}{}

\position{Graduate Research Assitant}
\address{707 Appalachian Dr.~Apt 10, Blacksburg, 24060}

\mobile{+1 540 257 2206}
\email{\href{mailto:jacky92@vt.edu}{\nolinkurl{jacky92@vt.edu}}}
\github{diogocamacho}
\linkedin{diogocamacho}
\twitter{DiogoMCamacho}

% \gitlab{gitlab-id}
% \stackoverflow{SO-id}{SO-name}
% \skype{skype-id}
% \reddit{reddit-id}

\quote{Mathematical Modeling, Data Analytics, System Biology}

\usepackage{booktabs}

\providecommand{\tightlist}{%
	\setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

%------------------------------------------------------------------------------



% Pandoc CSL macros
\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
\newlength{\csllabelwidth}
\setlength{\csllabelwidth}{3em}
\newenvironment{CSLReferences}[3] % #1 hanging-ident, #2 entry spacing
 {% don't indent paragraphs
  \setlength{\parindent}{0pt}
  % turn on hanging indent if param 1 is 1
  \ifodd #1 \everypar{\setlength{\hangindent}{\cslhangindent}}\ignorespaces\fi
  % set entry spacing
  \ifnum #2 > 0
  \setlength{\parskip}{#2\baselineskip}
  \fi
 }%
 {}
\usepackage{calc}
\newcommand{\CSLBlock}[1]{#1\hfill\break}
\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\csllabelwidth}{#1}}
\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth - \csllabelwidth}{#1}}
\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}

\begin{document}

% Print the header with above personal informations
% Give optional argument to change alignment(C: center, L: left, R: right)
\makecvheader

% Print the footer with 3 arguments(<left>, <center>, <right>)
% Leave any of these blank if they are not needed
% 2019-02-14 Chris Umphlett - add flexibility to the document name in footer, rather than have it be static Curriculum Vitae
\makecvfooter
  {January 2022}
    {Xiangyu Yao~~~Â·~~~Curriculum Vitae}
  {\thepage}


%-------------------------------------------------------------------------------
%	CV/RESUME CONTENT
%	Each section is imported separately, open each file in turn to modify content
%------------------------------------------------------------------------------



\hypertarget{qualifications-and-research-interests}{%
\section{Qualifications and research
interests}\label{qualifications-and-research-interests}}

Highly effective computational system biologist, with graduate and
post-doctoral work focusing on network inference and machine learning.
Industry experience in development and implementation of computational
tools for multi omics data analysis (including next-generation
sequencing, metabolomics, proteomics), drug discovery, and target
identification. I am interested in the application of machine
learning/deep learning tools and techniques in the context of drug
discovery, disease biology characterization, large data analytics for
biology, while focused on bridging the gap between the computational and
experimental labs through highly engaging and fruitful collaborations.

\hypertarget{technical-skills}{%
\section{Technical Skills}\label{technical-skills}}

Machine learning, multi omics data analytics, R/Bioconductor, keras,
perl, Latex, python, MATLAB, awk, bash. Adept user of OS X/macOS, Unix.
Familiarity with cloud computing architectures (AWS) and high
performance computing environments.

\hypertarget{education}{%
\section{Education}\label{education}}

\textbf{{Virginia Polytechnic Institute and State University}}
\hfill \emph{Blacksburg, VA}\\
Ph.D.~in Genetics, Bioinformatics, and Computational Biology
\hfill \emph{2007}

\textbf{{Faculdade de Ciencias da Universidade de Lisboa}}
\hfill \emph{Lisboa, Portugal}\\
B. Sc. in Biochemistry \hfill \emph{2002}

\hypertarget{experience}{%
\section{Experience}\label{experience}}

\textbf{{Wyss Institute @ Harvard University}} \hfill \emph{Boston,
MA}\\
Lead, Predictive BioAnalytics Initiative, Advanced Technology Team
\hfill \emph{July 2016 - Present}

I am currently the group lead of the Predictive BioAnalytics Initiative
at the Wyss Institute, where I lead a team of computational biologists,
computer scientists, and software engineers to address challenges in
machine learning and biomedical data sciences, both in-house or thtough
external collaborations. Some of the functions associated with the role
include:

\begin{itemize}
\tightlist
\item
  Development and implementation of research strategy for the
  Initiative, focusin on enabling ML/DL/AI capabilities
\item
  Managing and mentoring staff scientists, post-doctoral fellows,
  graduate students, and interns
\item
  Writing and managing federal grant applications and grant awards with
  DARPA, NIH
\item
  Hands-on development of algorithms and computational approaches for
  dissemination internally and with corporate partners via R Shiny
  applications
\item
  Responsible for the implementation of diverse tools for analysis of
  high throughput data (transcriptomics, RNA-seq, metabolomics,
  proteomics, 16S rDNA sequencing) in support of diverse grant work
\end{itemize}

\textbf{{Evelo Biosciences}} \hfill \emph{Cambridge, MA}\\
Senior Scientist, Computational Systems Biology Lead
\hfill \emph{January 2015 - April 2016}

As the lead Computational Systems Biologist at Evelo Biosciences, I was
involved in the build out of the computational capabilities of the
company, in support of preclinical development of microbiome-focused
therapeutics for oncology. Some of the functions of the role included:

\begin{itemize}
\tightlist
\item
  Implementation of diverse tools for analysis of high throughput data
  (transcriptomics, RNA-seq, metabolomics) as well as a 16S rDNA
  sequencing analysis pipeline
\item
  Responsible for the identification of novel therapeutic opportunities
  based on in-house and publicly available data
\item
  Implementation of analytical software tools to be used by bench
  scientists in R Shiny
\item
  Responsible for the interface with IT provider to delineate and expand
  computational capabilities of the company, from general to research
  needs
\end{itemize}

\textbf{{Ember Therapeutics}} \hfill \emph{Cambridge, MA}\\
Principal Scientist, Computational Systems Biology Lead
\hfill \emph{January 2014 - December 2014}

At Ember Therapeutics I was the Computational Systems Biology lead,
where I oversaw the implementation and deployment of novel analytical
tools for the identification of novel target opportunities for increased
energy expenditure, not only from publicly available data but also from
data coming from internal efforts. Some of the functions of the role
included:

\begin{itemize}
\tightlist
\item
  Responsible for the implementation of diverse tools for analysis of
  high throughput data (transcriptomics, RNA-seq, metabolomics)
\item
  Implementation of a knowledge based and data driven construction of
  screening libraries for recombinant proteins, small peptides, and RNAi
  therapeutic efforts
\item
  Implementation of analytical strategies (QC, statistical analyses,
  hit-calling) for high throughput screens
\item
  Responsible for the interface with IT provider to delineate and expand
  computational capabilities of the company, from general to research
  needs
\end{itemize}

\textbf{{Pfizer, Inc}} \hfill \emph{Cambridge, MA}\\
Senior Research Scientist, Computational Sciences Center of Emphasis
\hfill \emph{January 2011 - January 2014}

I was a member of the Computational Sciences Center of Emphasis at
Pfizer, providing computational support across different preclinical
programs at the organization, from cardiovascular disease to pain
management and drug repositioning efforts. Some of the functions of the
role included:

\begin{itemize}
\tightlist
\item
  Development and implementation of a network analysis tool for the
  characterization of differential networks in healthy and diseased
  populations under the scope of metabolic diseases
\item
  Implementation of network inference tools for the analysis of
  large-scale data sets
\item
  Development and implementation of a methodology for metabolite set
  enrichment analysis for metabolomics data
\item
  Performed data analysis in transcriptomics, proteomics and
  metabolomics for different partners within the organization with
  particular emphasis in Cardiovascular and Metabolic diseases
\end{itemize}

\textbf{{Howard Hughes Medical Institute @ Boston University}}
\hfill \emph{Boston, MA}\\
Post-doctoral Fellow \hfill \emph{July 2007 - January 2011}

Post-doctoral training with Dr.~James Collins at Boston University,
focusing on the application of machine learning and network inference
approaches in biomedicine. Some efforts included:

\begin{itemize}
\tightlist
\item
  Developed a network inference algorithm for identification of
  regulatory architectures of pathways
\item
  Identified a novel mechanism of action for antifungal drugs using
  transcriptomics and metabolomics data
\item
  Identified and characterized the small RNA regulatory network in
  bacterial systems using gene expression data
\item
  Performed data analysis of gene expression data and metabolomics data
  in bacterial and fungal systems
\end{itemize}

\hypertarget{awards-and-grants}{%
\section{Awards and Grants}\label{awards-and-grants}}

\textbf{{Molecular circuits in the hematopoietic stem cell niche (NIH)}}
\hfill \emph{\$415,000}\\
co-Investigator \hfill \emph{9/1/20}

\textbf{{Synergistic Discovery and Design (DARPA)}}
\hfill \emph{\$2,000,000}\\
co-PI \hfill \emph{9/1/17}

\hypertarget{publications}{%
\section{Publications}\label{publications}}

\begin{itemize}
\tightlist
\item
  Bojar, D, Powers, RK, \textbf{Camacho, DM}, Collins, JJ (2020), Deep
  learning reveals glycan-mediated host-microbe interactions, \emph{Cell
  Host \& Microbe} (\emph{accepted})
\item
  Valeri, J, Collins, KM, Ramesh, P, Alcantar, M, Lepe, BA, Lu, TK,
  \textbf{Camacho, DM} (2020), Sequence-to-function deep learning
  frameworks for engineered riboregulators, \emph{Nature communications}
  (\emph{accepted})
\item
  Jalili-Firoozinezhad, S, Gazzaniga, FS, Calamari, EL, \textbf{Camacho,
  DM}, Fadel, C, Nestor, B, Cronce, MJ, Tovaglieri, A, Levy, O, Gregory,
  KE, Breault, DT, Cabral, JMS, Kasper, DL, Novak, R, Ingber, DE (2019),
  A complex human gut microbiome cultured in an anaerobic
  intestine-on-a-chip, \emph{Nature Biomedical Engineering}, 3, 520-531
  {[}\href{https://www.ncbi.nlm.nih.gov/pubmed/31086325}{PubMed}{]}
\item
  Tovaglieri, A, Sontheimer-Phelps, A, Geirnaert, A, Prantil-Baun, R,
  \textbf{Camacho, DM}, Chou, DB, Jalili-Firoozinezhad, S, de Wouters,
  T, Kasendra, M, Super, M, Cartwright, M, Richmond, CA, Breault, DT,
  Lacroix, C, Ingber, DE (2019), Species-specific enhancement of
  enterohemorrhagic \emph{E. coli} pathogenesis mediated by microbiome
  metabolites, \emph{Microbiome}, 7, 43
  {[}\href{https://www.ncbi.nlm.nih.gov/pubmed/30890187}{PubMed}{]}
\item
  \textbf{Camacho, DM}, Collins, KM, Powers, RK, Costello, JC, Collins,
  JJ (2018), Next-generation machine learning for biological networks,
  \emph{Cell}, 173, 1581-1592
  {[}\href{https://www.ncbi.nlm.nih.gov/pubmed/29887378}{PubMed}{]}
\item
  Musah, S, Dimitrakakis, N, \textbf{Camacho, DM}, Church, GM, Ingber,
  DE (2018), Directed differentiation of human induced pluripotent stem
  cells into mature kidney podocytes and establishment of a Glomerulus
  Chip, \emph{Nature Protocols}, 13, 1662-1685
  {[}\href{https://www.ncbi.nlm.nih.gov/pubmed/29995874}{PubMed}{]}
\item
  Paandey, SP, Winkler, JA, Li, H, \textbf{Camacho, DM}, Collins, JJ,
  Walker, GC (2014), Central role for RNase YbeY in Hfq-dependent and
  Hfq-independent small-RNA regulation in bacteria, \emph{BMC Genomics},
  15, 121
  {[}\href{https://www.ncbi.nlm.nih.gov/pubmed/24511998}{PubMed}{]}
\item
  Galagan, JE, Minch, K, Peterson, M, Lyubetskya, A, Azzizi, E, Sweet,
  L, Gomes, A, Rustad, T, Dolganov, G, Glotova, I, Abeel, T, Mawhinney,
  C, Kennedy, A, Allard, R, Brabant, W, Krueger, A, Jaini, S, Honda, B,
  Yu, W-H, Hickey, M, Zucker, J, Garay, C, Weiner, B, Sisk, P, Stolte,
  C, Winkler, J, Van de Peer, Y, Iazzetti, P, \textbf{Camacho, D},
  Dreyfuss, J, Liu, Y, Dorhoi, A, Mollenkopf, H-J, Drogaris, P,
  Lamontagne, J, Zhou, Y, Piquenot, J, Park, ST, Raman, S, Kaufmann, S,
  Mohney, R, Chelsky, D, Moody, B, Sherman, D, Schoolnik, G (2013), The
  Mycobacterium tuberculosis regulatory network and hypoxia,
  \emph{Nature}, 499, 178-183
  {[}\href{https://www.ncbi.nlm.nih.gov/pubmed/23823726}{PubMed}{]}
\item
  Belenky, P, \textbf{Camacho, D}, Collins, JJ (2013), Fungicidal drugs
  induce a common oxidative-damage cellular death pathway, \emph{Cell
  Reports}, 3, 350-358
  {[}\href{https://www.ncbi.nlm.nih.gov/pubmed/23416050}{PubMed}{]}
\item
  Marbach, D, Costello, JC, Kuffner, R, Vega, N, Prill, RJ,
  \textbf{Camacho, DM}, Allison, KR, the DREAM5 Consortium, Kellis, M,
  Collins, JJ, Stolovitzky, G (2012), Wisdom of crowds for robust gene
  network inference, \emph{Nature Methods}, 9, 796-804
  {[}\href{https://www.ncbi.nlm.nih.gov/pubmed/22796662}{PubMed}{]}
\item
  Dwyer, DJ, \textbf{Camacho, DM}, Callura, JM, Kohanski, MA, Collins,
  JJ (2011), Antibiotic-induced bacterial cell death exhibits
  physiological and biochemical hallmarks of apoptosis, \emph{Molecular
  Cell}, 46, 561-572
  {[}\href{https://www.ncbi.nlm.nih.gov/pubmed/22633370}{PubMed}{]}
\item
  Modi, SR, \textbf{Camacho, DM}, Kohanski, MA, Collins, JJ (2011),
  Functional characterization of bacterial sRNAs using a network biology
  approach, \emph{Proc. Natl. Acad. Sci. USA}, 108, 15522-15527
  {[}\href{https://www.ncbi.nlm.nih.gov/pubmed/21876160}{PubMed}{]}
\item
  \textbf{Camacho, DM}, Collins, JJ (2009), Systems biology strikes
  gold, \emph{Cell}, 137, 24-26
  {[}\href{https://www.ncbi.nlm.nih.gov/pubmed/19345182}{PubMed}{]}
\item
  \textbf{Camacho, D}, Vera-Licona, P, Laubenbacher, R, Mendes, P
  (2007), Comparison of existing reverse engineering methods by use of
  an in silico system, \emph{Ann. N. Y. Acad. Sci.}, 1115, 73-89
  {[}\href{https://www.ncbi.nlm.nih.gov/pubmed/17925358}{PubMed}{]}
\item
  Mendes, P, \textbf{Camacho, D}, de la Fuente, A (2005), Modelling and
  simulation for metabolomics data analysis, \emph{Biochem. Soc.
  Trans.}, 33, 1427-1429
  {[}\href{https://www.ncbi.nlm.nih.gov/pubmed/16246137}{PubMed}{]}
\item
  \textbf{Camacho, D}, de la Fuente, A, Mendes, P (2005), The origin of
  correlations in metabolomics data, \emph{Metabolomics}, 1, 53-63
  {[}\href{https://link.springer.com/article/10.1007/s11306-005-1107-3}{Link}{]}
\item
  Martins, AM, \textbf{Camacho, D}, Shuman, J, Sha, W, Mendes, P,
  Shulaev, V (2004), A systems biology study of two distinct growth
  phases of \emph{Saccharomyces cerevisiae} cultures, \emph{Curr.
  Genomics}, 5, 649-663
  {[}\href{https://www.ingentaconnect.com/content/ben/cg/2004/00000005/00000008/art00005}{Link}{]}
\end{itemize}

\hypertarget{patents}{%
\section{Patents}\label{patents}}

\begin{itemize}
\tightlist
\item
  Soenksen, LR, \textbf{Camacho, DM}, Collins, KM, Angenent-Mari, NM,
  Garruss, AS, Collins, JJ, Church, GM, Method for deep learning for RNA
  synthetic biology (\emph{submitted})
\item
  Collins, JJ, Belenky, P, \textbf{Camacho, DM} (2015), Compositions and
  methods for treating fungal infections, WO2014130922A1
  {[}\href{https://patents.google.com/patent/WO2014130922A1/en}{Link}{]}
\end{itemize}

\hypertarget{references}{%
\section{References}\label{references}}

References will be provided upon request.



\end{document}
